### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by boecklim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ This is a TypeScript library for the Bitcoin ARC project.
...
 ```bash
-npm install js-arc-client
+npm install @bitcoin-a/arc-client
 ```
 
 ## Usage
 
 ```javascript
-import { ArcClient } from 'js-arc-client';
+import { ArcClient } from '@bitcoin-a/arc-client';
```
Comment (sirdeggen) [inline README.md:12]: Let's move them to @bsvb - it's cleaner, and we're changing brand to bsv blockchain association - bitcoin-a doesn't really roll off the tongue or fingers.

**Review by sirdeggen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (boecklim) [reply README.md:12]: OK, yes that makes sense.

File: README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ const result = await arcClient.getTransactionStatus(txid);
 or as commonjs module:
 
 ```javascript
-const { ArcClient } = require('js-arc-client').ArcClient;
+const { ArcClient } = require('@bitcoin-a/arc-client').ArcClient;
```
Comment (sirdeggen) [inline README.md:25]: Looks like we also need to remove one of two things here:

1. The {} braces around ArcClient on the left. OR
2. the .ArcClient at the end of this line.

Comment (sirdeggen) [reply README.md:25]: At the moment ArcClient is left undefined because it's being assigned to `exports.ArcClient.ArcClient`

**Comment by sirdeggen** [comment]
Closing because we'll hopefully be able to hand this to a js libraries team soon™.

**Pull request closed without merging**